packages/terra-application-utility/src/menu/ApplicationMenuUtility.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
```

Comment:
Are we expecting the MenuUtility and HeaderUtility to diverge at some point? It appears to be a 1-to-1 implementation with a different icon.